src/run_glue_no_trainer.py

```diff
@@ -0,0 +1,461 @@
+# coding=utf-8
```

Comment:
This file should be removed afterward :-)

Reply:
will remove.